### ISSUE TITLE
86Box Auto Updater

### ISSUE THREAD

**Issue by Slo2018** [86Box/86Box#3141]
Please, add Auto Updater in 86Box.

**Comment by OBattler** [comment]
I'm quite sure this is planned as part of the built-in manager that's being worked on.

**Comment by jadasse** [comment]
> I'm quite sure this is planned as part of the built-in manager that's being worked on.

https://github.com/86Box/86BoxManager/issues/90 yep

**Comment by OBattler** [comment]
No, that's not the planned built-in manager, but the old external manager by daviunic.

**Comment by jadasse** [comment]
> No, that's not the planned built-in manager, but the old external manager by daviunic.

There's a new manager?
